docs/source/pages/integrating_depthai_into_products.rst

```diff
@@ -0,0 +1,60 @@
+.. _integrating_depthai_into_products:
+
+Integrating DepthAI into products
+=================================
+
+How hard is to integrate DepthAI into our product
+*************************************************
+
+Not hard at all, since our platform is designed from the grounds up to achieve simple integration with
+customers products.
+
+What is SoM
+***********
+
+SoM stands for System on Module. In general it is just a small computer used to solve a specific task.
```

Comment:
small computer? Specify that there's a VPU (Myriad X, quick description about it) + electronics to power it (+ RAM / potentially flash